plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotIntegrationConnectorSmokeTest.java

```diff
@@ -137,6 +141,30 @@ protected String getPinotImageName()
         return PINOT_PREVIOUS_IMAGE_NAME;
     }
 
+    @Override
+    @BeforeClass
+    public void init()
+            throws Exception
+    {
+        super.init();
+        // Ensure two tables are available in Pinot with expected number of rows.
+        LOGGER.info("Waiting for up to 60 secs for Pinot to load required data set.");
+        QueryRunner queryRunner = getQueryRunner();
+        MaterializedResult regionRows = queryRunner.execute("SELECT * FROM tpch.tiny.region");
+        MaterializedResult nationRows = queryRunner.execute("SELECT * FROM tpch.tiny.nation");
+
+        assertQueryEventually(
```

Comment:
What happens if we add a new table and the first test uses it? I guess it may lead to failures.

Reply:
Right now test created all the tables in the init method, so I think it's in general ok to just check the last tables.

Reply:
https://github.com/trinodb/trino/issues/14277#issue-1384725737 message contains my_table, not region nor nation.  Verifying last tables won't resolve the flaky issue in my opinion.

Reply:
true, added more tables.